components/servo/tests/webview.rs

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+//! WebView API unit tests.
+//!
+//! Since all Servo tests must rust serially on the same thread, it is important
```

Comment:
```suggestion
//! Since all Servo tests must run serially on the same thread, it is important
```

Reply:
Thanks! I also need to debug exactly why shutdown is failing so badly. In the meantime I'll move this change back into the draft phase.